src/common/utils/getViewer.ts

```diff
@@ -47,6 +49,10 @@ export const getUserGroup = ({
   } catch (error) {
     logger.error(error)
   }
+
+  if (role === USER_ROLE.admin) {
+    return 'b'
```

Comment:
any related discussion on this change?

Reply:
Yes, @jeremyOk proposed this at wrote the logic in the SQL which is used in migrations: https://github.com/thematters/matters-server/pull/1538/files#diff-b726e1a846dbc8b25876ac766b0ceaa41f4983702650e221ea1a80e84a876a11R23